cellfinder_napari/detect.py

```diff
@@ -129,183 +80,130 @@ def widget(
         detection_label,
         data_options,
         viewer: napari.Viewer,
-        Signal_image: napari.layers.Image,
-        Background_image: napari.layers.Image,
+        signal_image: napari.layers.Image,
+        background_image: napari.layers.Image,
         voxel_size_z: float,
         voxel_size_y: float,
         voxel_size_x: float,
         detection_options,
-        Soma_diameter: float,
+        soma_diameter: float,
         ball_xy_size: float,
         ball_z_size: float,
-        Ball_overlap: float,
-        Filter_width: float,
-        Threshold: int,
-        Cell_spread: float,
-        Max_cluster: int,
+        ball_overlap: float,
+        filter_width: float,
+        threshold: int,
+        cell_spread: float,
+        max_cluster: int,
         classification_options,
-        Trained_model: Path,
+        trained_model: Path,
         misc_options,
-        Start_plane: int,
-        End_plane: int,
-        Number_of_free_cpus: int,
-        Analyse_local: bool,
-        Debug: bool,
+        start_plane: int,
+        end_plane: int,
+        number_of_free_cpus: int,
+        analyse_local: bool,
+        debug: bool,
         reset_button,
     ) -> List[napari.types.LayerDataTuple]:
```

Comment:
In `python>=3.9`, potentially don't change now but this is the future! 🚀 

```suggestion
    ) -> list[napari.types.LayerDataTuple]:
```

Reply:
Yup!